frontend/src/screens/wallet/send/Onchain.tsx

```diff
@@ -0,0 +1,360 @@
+          },
+          body: JSON.stringify({
+            toAddress: address,
+            amount: +amount,
+            feeRate: +feeRate,
+          }),
+        }
+      );
+      if (!response?.txId) {
+        throw new Error("No address in response");
+      }
+      navigate(`/wallet/send/onchain-success`, {
+        state: {
+          amount: +amount,
+          txId: response.txId,
+        },
+      });
+      toast({
+        title: "Successfully broadcasted transaction",
+      });
+    } catch (e) {
+      toast({
+        variant: "destructive",
+        title: "Failed to send payment",
+        description: "" + e,
+      });
+      console.error(e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  if (!balances || (!recommendedFees && !mempoolError)) {
+    return <Loading />;
+  }
+
+  return (
+    <form onSubmit={onSubmit} className="grid gap-6">
+      <div className="grid gap-2">
+        <Label htmlFor="amount">Amount</Label>
+        <Input
+          id="amount"
+          type="number"
+          value={amount}
+          placeholder="Amount in Satoshi..."
+          onChange={(e) => {
+            setAmount(e.target.value.trim());
+          }}
+          min={ONCHAIN_DUST_SATS}
+          max={balances.onchain.spendable}
+          required
+          autoFocus
+          className="[appearance:textfield] [&::-webkit-outer-spin-button]:appearance-none [&::-webkit-inner-spin-button]:appearance-none"
+          endAdornment={
+            <FormattedFiatAmount amount={Number(amount)} className="mr-2" />
+          }
+        />
+        <div className="flex justify-between text-muted-foreground text-xs sensitive slashed-zero">
+          <div>
+            On-chain Balance:{" "}
+            {new Intl.NumberFormat().format(balances.onchain.spendable)} sats
+          </div>
+          <FormattedFiatAmount
+            className="text-xs"
+            amount={balances.onchain.spendable}
+          />
+        </div>
+      </div>
+      <div className="flex items-center justify-between">
+        <Label htmlFor="swap" className="font-medium text-sm cursor-pointer">
+          Swap from Spending Balance
+        </Label>
+        <Switch id="swap" onCheckedChange={setSwap} />
+      </div>
+      <div className="grid gap-2 text-sm border-t pt-4">
+        <div className="flex items-center justify-between">
+          <p className="text-muted-foreground">On-chain Fee</p>
+          {feeRate ? <p>{feeRate} sat/vB</p> : <Loading className="w-4 h-4" />}
+        </div>
+      </div>
+      {amount && +amount < 10_000 && (
+        <Alert>
+          <InfoIcon className="h-4 w-4" />
+          <AlertTitle>Amount not ideal for On-chain transaction</AlertTitle>
+          <AlertDescription>
+            Small amounts can become unspendable during fee spikes. Consider
+            using Lightning instead.
+          </AlertDescription>
+        </Alert>
+      )}
+      <div className="flex gap-2">
```

Comment:
We need an alert about spending anchor reserves too. Can you make sure we have the same functionality as the existing redeem onchain funds page?